history.txt

```diff
@@ -1,6 +1,42 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
+-  improved UI experience when moving annotated data between groups in
+   OMERO.web
+-  major improvement in loading annotations in the righthand panel of
+   OMERO.web
+-  much better handling of ROIs covering large planes in OMERO.insight
+-  rendering setting fixes for copy and paste actions
+-  rendering fixes for floating point data
+-  image interpolation can now be turned on and off in the clients
+-  more readable timestamps
```

Comment:
Maybe "Better formatting of Delta-T and exposure times in Insight and web client"